src/cds_objects.h

```diff
@@ -292,7 +292,7 @@ class CdsObject {
     }
 
     /// \brief Get number of resource tags
-    int getResourceCount() const { return resources.size(); }
+    int getResourceCount() const { return int(resources.size()); }
```

Comment:
try to change the return type to size_t

Reply:
I had no idea why this returned int. Hence I decided to be conservative.

Reply:
I prefer going forward!